packages/create-sitecore-jss/src/templates/nextjs/src/lib/page-props.ts

```diff
@@ -6,13 +6,19 @@ import {
 } from '@sitecore-jss/sitecore-jss-nextjs';
 
 /**
- * Sitecore page props
+ * Nextjs app props
  */
-export type SitecorePageProps = {
+export type NextAppProps = {
```

Comment:
I don't understand the need for `NextAppProps`. Why not use what we have currently for `SitecorePageProps`? Seems like `AppProps<T>` should use your page props type (i.e. this is the type off `AppProps.pageProps`).

Reply:
@ambrauer Not really, I tried to log `pageProps` and inside the `_app.tsx` it doesn't contain `notFound`, `redirect`